tracer/src/Datadog.Trace/ServiceFabric/ServiceRemotingClient.cs

```diff
@@ -67,7 +67,7 @@ private static void ServiceRemotingClientEvents_SendRequest(object? sender, Even
                         SpanContextPropagator.Instance.Inject(
                             span.Context,
                             messageHeaders,
-                            &HeadersSetter);
+                            (headers, headerName, headerValue) => headers.TryAddHeader(headerName, headerValue));
```

Comment:
```suggestion
                            static (headers, headerName, headerValue) => headers.TryAddHeader(headerName, headerValue));
```

Reply:
Added in 31aee89. I cannot see any changes in the codegen as expected.